packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -226,22 +225,6 @@ class _InspectorTreeState extends State<InspectorTree>
     _focusNode.requestFocus();
   }
 
-  void _onScrollYChange() {
```

Comment:
I agree the existing way this was done should be improved but doesn't removing this mean that we will end up scrolled to a X coordinate that doesn't show any useful content when we scroll to a new Y coordinate?

Reply:
> we will end up scrolled to a X coordinate that doesn't show any useful content when we scroll to a new Y coordinate?

Yes - we will; the user will have to manually scroll around. We could improve things here, but the current behavior of very jerky vertical scrolling is IMO more of a friction for users than having automatic horizontal scrolling.

I could keep this commented out and have a todo? Add a todo, and reference this commit (to find the source later)?

Reply:
commented out with a todo sounds good. I don't think the right fix will be hard. we just need some logic layered on top of the scroll controller to smooth things out.

Reply:
Done! Added back in, commented out, with a comment referencing this PR.